framework/behaviors/BlameableBehavior.php

```diff
@@ -96,9 +96,9 @@ public function init()
      */
     protected function getValue($event)
     {
-        if ($this->value === null) {
-            $user = Yii::$app->get('user', false);
-            return $user && !$user->isGuest ? $user->id : null;
+        if ($this->value === null && Yii::$app->has('user')) {
+            $user = Yii::$app->get('user');
+            return !$user->isGuest ? $user->id : null;
```

Comment:
this equal:
return $user->getId();
https://github.com/yiisoft/yii2/blob/master/framework/web/User.php#L338